lib/ipinfo/cache/default_cache.rb

```diff
@@ -0,0 +1,23 @@
+require 'ipinfo/cache/cache_interface'
+require 'lrucache'
+
+module IPinfo
+  class DefaultCache < CacheInterface
+
+    def initialize(ttl, max_size)
+      @cache = LRUCache.new(:ttl => ttl, :max_size => max_size)
+    end
+
+    def get(key)
+      @cache[key]
+    end
+
+    def set(key, value)
+      @cache[key] = value
+    end
+
+    def contains(key)
```

Comment:
this returns boolean, so method should end with question mark. e.g. contains? 

https://github.com/rubocop-hq/ruby-style-guide#bool-methods-prefix